addons/product/models/product_pricelist.py

```diff
@@ -302,6 +302,8 @@ def _price_get_multi(self, pricelist, products_by_qty_by_partner):
             list(pycompat.izip(**products_by_qty_by_partner)))
 
     def _get_partner_pricelist(self, partner_id, company_id=None):
+        # TODO (review): Deprecated? At least not used since https://github.com/odoo/odoo/commit/d39d583b213af02bad12721d9c610def6cb09161
```

Comment:
to mark as deprecated + nextActivity oto clean in it in master after fwdport

Reply:
Done